src/components/common/DocumentModal/DocumentModal.tsx

```diff
@@ -82,7 +83,7 @@ export const DocumentModal: React.FC<DocumentModalProps> = ({
                 <span className="slightly-smaller-text"> of {numPages}</span>
               </div>
             )}
-            {fileType !== 'other' && (
+            {fileType !== 'other' && !preventDownload && (
```

Comment:
`preventDownload` hides the option to download or display file in new tab. This is used for Previews, as we don't want preview files stored on users' computers for the most part.